scripts/auto_sshfs_mount.py

```diff
@@ -12,7 +12,7 @@
 
 #==============================================================================
 
-def Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
+def _Mount(local, remote, ServerAliveInterval=20, ServerAliveCountMax=3):
```

Comment:
In Python convention CamelCase is for classes. Function and variable names should use lower case with underscores. Should be: `def _mount(local, remote, server_alive_interval=20, server_alive_count_max=3):`